src/Application/HttpModels/PaymentMethodType.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Application\HttpModels;
+
+enum PaymentMethodType
+{
+    case card;
```

Comment:
(copied from commit comments)  can we name the cases in PascalCase? I went with that for the status because it's recommended in the PER standard.